packages/@expo/cli/src/install/installAsync.ts

```diff
@@ -94,11 +96,69 @@ export async function installPackagesAsync(
     }using {bold ${packageManager.name}}`
   );
 
-  await packageManager.addWithParametersAsync(versioning.packages, packageManagerArguments);
+  await packageManager.addAsync([...packageManagerArguments, ...versioning.packages]);
 
   await applyPluginsAsync(projectRoot, versioning.packages);
 }
 
+export async function fixPackagesAsync(
+  projectRoot: string,
+  {
+    packages,
+    packageManager,
+    sdkVersion,
+    packageManagerArguments,
+  }: {
+    packages: Awaited<ReturnType<typeof getVersionedDependenciesAsync>>;
+    /** Package manager to use when installing the versioned packages. */
+    packageManager: PackageManager.NodePackageManager;
+    /**
+     * SDK to version `packages` for.
+     * @example '44.0.0'
+     */
+    sdkVersion: string;
+    /**
+     * Extra parameters to pass to the `packageManager` when installing versioned packages.
+     * @example ['--no-save']
+     */
+    packageManagerArguments: string[];
+  }
+): Promise<void> {
+  if (!packages.length) {
+    return;
+  }
+
+  const { dependencies = [], devDependencies = [] } = groupBy(packages, (dep) => dep.packageType);
+  const versioningMessages = getOperationLog({
+    othersCount: 0, // All fixable packages are versioned
+    nativeModulesCount: packages.length,
+    sdkVersion,
+  });
+
+  Log.log(
+    chalk`\u203A Installing ${
+      versioningMessages.length ? versioningMessages.join(' and ') + ' ' : ''
+    }using {bold ${packageManager.name}}`
+  );
+
+  if (dependencies.length) {
+    const versionedPackages = dependencies.map(
+      (dep) => `${dep.packageName}@${dep.expectedVersionOrRange}`
+    );
+
+    await packageManager.addAsync([...packageManagerArguments, ...versionedPackages]);
+
+    await applyPluginsAsync(projectRoot, versionedPackages);
```

Comment:
I kind of assumed it, but I don't think `applyPluginsAsync` needs to run for `devDependencies`, right? Maybe it doesn't even need to run for the `fixDependenciesAsync`, because it only "fixes DependencyIssue". That implies it's already there.

That being said, I just did it for consistency with `installDependenciesAsync`.